robottelo/vm.py

```diff
@@ -186,7 +186,8 @@ def create(self):
         result = ssh.command(
             u'for i in {{1..60}}; do ping -c1 {0}.local && exit 0; sleep 1;'
             u' done; exit 1'.format(self._target_image),
-            self.provisioning_server,
+            self.provisioning_server if self.bridge == 'br0'
+            else settings.server.hostname,  # in VLANs ping from SAT
```

Comment:
Think creating a variable is more readable:
```python
ping_from_host_name =  self.provisioning_server if self.bridge == 'br0' else settings.server.hostname
result = ssh.command(
              u'for i in {{1..60}}; do ping -c1 {0}.local && exit 0; sleep 1;'
              u' done; exit 1'.format(self._target_image),
              ping_from_host_name
)
```

Reply:
I read quite nice - ternary operator , but yeah C++ has it nicer :)

Used just once - no need to create var